bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+log = logging.getLogger(__name__)
+
+
+class PrideLeader(commands.Cog):
+    """Gives a Pride Leader Info."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.pride = self.load_json()
+
+    @staticmethod
+    def load_json() -> dict:
+        """Loads pride leader information from static json resource."""
+        explanation_file = Path("bot/resources/pride/prideleader.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            pride = json.load(json_data)
+
+        return pride
```

Comment:
I didn't see the method being used anywhere else, just do it all in the init. "Load_json" insn't really a descriptive name either. It could be any json

Reply:
its used in embed builder